lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -24,6 +24,10 @@ class Main < CC::Engine::Analyzers::Base
 
           def process_file(file)
             RubyParser.new.process(File.binread(file), file, TIMEOUT)
+          rescue Timeout::Error
+            warn "TIMEOUT parsing #{file}. Skipping."
+          rescue => e
```

Comment:
I don't remember the specifics anymore. It's been a while and I'm sleep deprived today. I can roll out that rescue and rerun my gauntlet if necessary.

Otherwise, I guess objections have been met and this should go in? Then I can clean up error handling.